tests/plugins/test_koji_promote.py

```diff
@@ -737,6 +741,10 @@ def validate_output(self, output, metadata_only):
                 digest_pullspec = image.to_str(tag=False) + '@' + fake_digest(image)
                 assert digest_pullspec in repositories_digest
 
+            tags = docker['tags']
+            assert isinstance(tags, list)
+            assert all(is_string_type(tag) for tag in tags)
```

Comment:
There is a separate test that actually tests for expected values. This 'validate' method is more about types.